docs/index.bs

```diff
@@ -2918,23 +2975,27 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
   </section>
 
   <section algorithm>
-    <h3 id="run-service-worker-algorithm"><dfn>Run Service Worker</dfn></h3>
+    <h3 id="setup-serviceworkerglobalscope"><dfn>Setup ServiceWorkerGlobalScope</dfn></h3>
 
       : Input
       :: |serviceWorker|, a [=/service worker=]
-      :: |forceBypassCache|, an optional boolean, false by default
       : Output
-      :: a [=Completion=] or *failure*
+      :: a boolean
 
-      Note: This algorithm blocks until the service worker is [=running=] or fails to start.
+      Note: This algorithm returns true if |serviceWorker| has a {{ServiceWorkerGlobalScope}} usable for a CSP check.
```

Comment:
I was not sure if {{ServiceWorkerGlobalScope}} may exist outside of service worker agent.  That is why I created it and attached it.
If we create {{ServiceWorkerGlobalScope}} every time, it should technically be independent {{ServiceWorkerGlobalScope}} from what is used for running the fetch handler.  Is it fine to do in the spec?

Reply:
If having a separate global scope from what is used for running the fetch handler is a problem, we could always branch/use the existing one if we already have one, and otherwise create a temporary one to be immediately thrown away afterwards. While a real implementation wouldn't do that, as mentioned earlier in your changes, a real implementation probably wouldn't even create a global scope at all in this code path.

Reply:
My question here is that:
1. if we can create a {{ServiceWorkerGlobalScope}} without agent.  It might be no.
2. Now we use [ServiceWorkerGlobalScope is ready flag](https://pr-preview.s3.amazonaws.com/yoshisatoyanagisawa/ServiceWorker/pull/1701.html#service-worker-serviceworkerglobalscope-is-ready-flag) to know if the {{ServiceWorkerGlobalScope}} has been set up in the agent.  However, if we do not have an associated flag, how can we know if the {{ServiceWorkerGlobalScope}} is ready?  Can {{ServiceWorkerGlobalScope}} has an associated flag for that purpose?

Reply:
I moved where the global object is set to avoid the issue.
Now, I removed the flag.